### ISSUE TITLE
Usage/config file documentation for modules

### ISSUE THREAD

**Issue by sjrct** [Halibot/halibot-legacy#21]
Documentation for how to configure individual modules would be nice, maybe it shouldn't be in the main readme though, probably should separate with the separation of the modules from this repo itself.

**Comment by richteer** [comment]
Yep, that's on its way. Waiting to complete #17 first in case an overhaul of the config format is deemed necessary, and then it will be in the new repo resolving #16
